spec/draper/decorator_spec.rb

```diff
@@ -596,12 +596,16 @@ module Draper
           expect(decorator.hello_world).to be :delegated
         end
 
-        it "adds delegated methods to the decorator when they are used" do
-          decorator = Decorator.new(double(hello_world: :delegated))
+        it "allows calling `super`" do
+          decorator_class = Class.new(Decorator) do
+            def hello_world
+              super and "overriden hello world"
+            end
+          end
+          decorator = Decorator.new(double(hello_world: "hello world"))
```

Comment:
You should use defined `decorator_class` here:

```ruby
decorator = decorator_class.new(double(hello_world: "hello world"))
```

After this change tests pass